letsencrypt-win-simple/Plugin/OverviewPlugin.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Serilog;
+using Microsoft.Web.Administration;
+
+namespace LetsEncrypt.ACME.Simple
+{
+    public class OverviewPlugin : Plugin
+    {
+        private const string ClientName = "letsencrypt-win-simple";
+
+        public override string Name => "OverviewPlugin";
+
+        public override void PrintMenu()
+        {
+            Console.WriteLine(" O: Get overview of all bindings, protocols, renewals and certificates");
+        }
+
+        public override void HandleMenuResponse(string response, List<Target> targets)
+        {
+            if (response == "o")
+            {
+                Console.WriteLine("Running Overview Plugin");
+                Log.Information("Running Overview Plugin");
+
+                GetIisSites();
+                Console.WriteLine();
+                Console.WriteLine("Press enter to continue.");
+                Console.ReadKey();
+                GetRenewals();
+                Console.WriteLine();
+                Console.WriteLine("Press enter to continue.");
+                Console.ReadKey();
+                GetCertificates();
+                Console.WriteLine();
+            }
+        }
+
+        public static void GetIisSites()
```

Comment:
GetSites dosent give me what I need! Also GetSites has logic which should not be there if it should be used elsewhere in the program.
This overview is exately what I need. Does it matter if you get the point? (I dont mean to be rude but your comment is a little condescending)
Pagination is in my humble view not appropiate in a console app. If you have too many sites to show you could increase the buffer with Console.SetBufferSize.